src/types/enums.ts

```diff
@@ -0,0 +1,5 @@
+export enum ROUTES {
+  AUTH_ERROR = 'auth-error',
+  INDEX = '/',
+  NOT_FOUND = '404',
+}
```

Comment:
Now I don't know if this has been the case in other projects, then it is fine, but having the routes under /types feels quite strange, since these don't really have much to do with types :D

Reply:
I was looking at https://github.com/City-of-Helsinki/parking-permits-admin-ui and this is how they have done it.

But I will change this for this project. As you said, it might be better somewhere else so I'll create a `constants.ts ` file for these